shuup/admin/settings.py

```diff
@@ -67,3 +67,10 @@
 #: menu item. Useful in case there is need to always open some
 #: certain menus.
 SHUUP_ALWAYS_ACTIVE_MENU_CATEGORY_IDENTIFIERS = []
+
+#: Indicates whether all object select fields should load options asynchronously.
+#: When True, fields will load options through AJAX requests instead
+#: of generating them while rendering the page. For enviroments with huge amount of options
+#: in fields, like categories, it is best to have this enabled.
+#:
+SHUUP_ADMIN_LOAD_SELECT_OBJECTS_ASYNC = True
```

Comment:
Could this be ("categories": True) thing where I can limit this per model or something similar. In this solution do we now loose quick adds. This does not seem right.